src/content/developers/docs/intro-to-ether/index.md

```diff
@@ -34,15 +34,15 @@ It is [common](https://www.reuters.com/article/us-crypto-currencies-lending-insi
 
 Minting is the process in which new ether gets created on the Ethereum ledger. The underlying Ethereum protocol creates the new ether, and it is not possible for a user to create ether.
 
-Ether is minted when a miner creates a block on the Ethereum blockchain. As an incentive to miners, the protocol grants a reward in each block, incrementing the balance of an address set by the block's miner. The block reward has changed over time, and today it is 2 ETH per block.
+Ether is minted when a new block is created on the Ethereum blockchain. As an incentive to build blocks, the protocol grants a reward in each block, incrementing the balance of an address set by the block producer. The block reward has changed over time, and today it is 2 ETH per block. After the merge issuance to each validator depends upon the amount of ether they have staked and their performance.
```

Comment:
Just noting ETH is also created on the Beacon Chain as rewards to validators for attesting (occurring now pre-merge, even if slots don't contain blocks)